eox_tenant/management/commands/change_domain.py

```diff
@@ -57,7 +57,7 @@ def handle(self, *args, **options):
         self.suffix_stage_ecommerce_domain = options['suffix_ecommerce_domain']
 
         # Changing microsites objects
-        for microsite in Microsite.objects.all():  # pylint: disable=no-member
+        for microsite in Microsite.objects.all():
```

Comment:
how did you make all this unnecessary on this PR?

Reply:
I deleted the _# pylint: disable=no-member_ because the quality test was failing [in circle CI](https://circleci.com/gh/eduNEXT/eox-tenant/1350#tests/containers/0) It [passed](https://circleci.com/gh/eduNEXT/eox-tenant/1351) after I deleted the pylint disabled.

Reply:
Do you have any recommendations?